yarn.lock

```diff
@@ -2612,6 +3647,20 @@
     "@polkadot/types" "^1.0.0-beta.18"
     "@polkadot/util-crypto" "^2.0.0-beta.5"
 
+"@polkadot/api@^1.0.0-beta.44":
```

Comment:
twice this package again

Reply:
i'll be more careful, haven't found a better way yet to manage these dependencies for dockerizing a single workspace